cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -252,83 +257,66 @@
 
 {% block javascript scoped %}
 
-<script>
-    {# Minified dynamic JavaScript loader that injects a script tag in the head
-       of the page. It also provides a callback parameter. #}
-    function loadScript(t,e){var n=document.createElement('script');n.type='text/javascript',n.onload=function(){return e?e():null},n.src=t,document.getElementsByTagName('head')[0].appendChild(n)}
-
-    {#
-      Check and include component-level JavaScript.
-      Component-level JavaScript adds scripts appearing in
-      unprocessed/js/routes/on-demand/[component].js
-      to the page, based on what components were added in Wagtail.
-    #}
-    function loadScriptComponent(){
-      {% if page and page.media %}
-         {% for js in page.media %}
-           loadScript('{{ static('js/routes/on-demand/' + js) }}');
-         {% endfor %}
-      {% endif %}
-    }
 
-    {#
-      TODO: Remove the below blocks when all pages
-            using this template are moved to Wagtail.
-            This moves us away from a URL-dependant script inclusion scheme.
-    #}
+{#
+  Modernizr adds a no-js class the HTML element if ECMAScript 5 features aren't
+  available. Check for that CSS class here and act accordingly.
+#}
+<script async>
+  if ( document.body.parentElement.className.indexOf( 'no-js' ) === -1 ) {
```

Comment:
Would `document.documentElement` or `document.children[0]` work here for the lookup? Would need to check IE9 compatibility.

Reply:
Or I guess IE10+